sport-moment-nft/contracts/EPL.cdc

```diff
@@ -0,0 +1,1124 @@
+                            )
+                        ]
+                    )
+                case Type<MetadataViews.NFTCollectionDisplay>():
+                    let bannerImage = MetadataViews.Media(
+                        file: MetadataViews.HTTPFile(
+                            url: "https://assets.eplonflow.com/static/epl-logos/EPL_Logo_Horizontal_B.png"
+                        ),
+                        mediaType: "image/png"
+                    )
+                    let squareImage = MetadataViews.Media(
+                        file: MetadataViews.HTTPFile(
+                            url: "https://assets.eplonflow.com/static/epl-logos/EPL_Logo_Primary_B.png"
+                        ),
+                        mediaType: "image/png"
+                    )
+                    return MetadataViews.NFTCollectionDisplay(
+                        name: "EPL On Flow",
+                        description: "Collect EPL's biggest Moments and get closer to the game than ever before",
+                        externalURL: MetadataViews.ExternalURL("https://eplonflow.com/"),
+                        squareImage: squareImage,
+                        bannerImage: bannerImage,
+                        socials: {
+                            "instagram": MetadataViews.ExternalURL(" https://instagram.com/eplonflow"),
+                            "twitter": MetadataViews.ExternalURL("https://twitter.com/EPLOnFlow"),
+                            "discord": MetadataViews.ExternalURL("https://discord.gg/EPLOnFlow"),
+                            "facebook": MetadataViews.ExternalURL("https://www.facebook.com/EPLOnFlow/")
+                        }
+                    )
+                case Type<MetadataViews.Royalties>():
+                    let royaltyReceiver: Capability<&{FungibleToken.Receiver}> =
+                        getAccount(EPL.RoyaltyAddress()).getCapability<&AnyResource{FungibleToken.Receiver}>(MetadataViews.getRoyaltyReceiverPublicPath())
+                    return MetadataViews.Royalties(
+                        royalties: [
+                            MetadataViews.Royalty(
+                                receiver: royaltyReceiver,
+                                cut: 0.05,
+                                description: "EPL marketplace royalty"
+                            )
+                        ]
+                    )
+            }
+
+            return nil
+        }
+    }
+
+    //------------------------------------------------------------
+    // Collection
+    //------------------------------------------------------------
+
+    /// A public collection interface that allows Moment NFTs to be borrowed
+    ///
+    pub resource interface MomentNFTCollectionPublic {
+        pub fun deposit(token: @NonFungibleToken.NFT)
+        pub fun batchDeposit(tokens: @NonFungibleToken.Collection)
+        pub fun getIDs(): [UInt64]
+        pub fun borrowNFT(id: UInt64): &NonFungibleToken.NFT
```

Comment:
Please also include `borrowNFTSafe()` in the public interface and the implementation:
https://github.com/onflow/flow-nft/blob/master/contracts/NonFungibleToken.cdc#L143

Reply:
`borrowMomentNFT` could even be removed now that we have the safe borrow

Reply:
Done.